frontend/src/features/mapSideBar/layout/MapSideBarLayout.tsx

```diff
@@ -1,115 +1,108 @@
-import { ReactComponent as LotSvg } from 'assets/images/icon-lot.svg';
-import classNames from 'classnames';
 import * as Styled from 'components/common/styles';
 import TooltipWrapper from 'components/common/TooltipWrapper';
 import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
 import { FaWindowClose } from 'react-icons/fa';
 import VisibilitySensor from 'react-visibility-sensor';
 import styled from 'styled-components';
 
 interface IMapSideBarLayoutProps {
-  show: boolean;
+  showSideBar: boolean;
   setShowSideBar: (show: boolean) => void;
   title: React.ReactNode;
-  hidePolicy?: boolean;
-  /** property name for title */
-  propertyName?: string;
-  header: React.ReactNode;
+  header?: React.ReactNode;
+  icon: React.ReactNode | React.FunctionComponent;
+  showCloseButton?: boolean;
 }
 
 /**
  * SideBar layout with control bar and then form content passed as child props.
  * @param param0
  */
 const MapSideBarLayout: React.FunctionComponent<IMapSideBarLayoutProps> = ({
-  show,
+  showSideBar,
   setShowSideBar,
-  hidePolicy,
   title,
-  propertyName,
+  header,
+  icon,
+  showCloseButton,
   ...props
 }) => {
   return (
-    <StyledMapSideBarLayout
-      className={classNames('map-side-drawer', {
-        show: show,
-      })}
-    >
+    <StyledMapSideBarLayout show={showSideBar}>
       <VisibilitySensor partialVisibility={true}>
         {({ isVisible }: any) => (
           <>
             <TitleBar>
-              <Underline>
-                <LotIcon className="mr-1" />
-                <Styled.H1 className="mr-auto">{title}</Styled.H1>
+              <Row>
+                <Col>
+                  <Styled.H1 className="mr-auto">
+                    {icon}
+                    {title}
+                  </Styled.H1>
+                </Col>
 
-                <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
-                  <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
-                </TooltipWrapper>
-              </Underline>
+                {showCloseButton && (
+                  <Col xs="auto">
+                    <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
+                      <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
+                    </TooltipWrapper>
+                  </Col>
+                )}
+              </Row>
+              <Underline />
             </TitleBar>
-            <Header>{props.header}</Header>
-            <Content>{isVisible ? props.children : null}</Content>
+
+            <StyledBody>
+              {header && isVisible && <Header>{header}</Header>}
+              <Content>{isVisible ? props.children : null}</Content>
+            </StyledBody>
           </>
         )}
       </VisibilitySensor>
     </StyledMapSideBarLayout>
   );
 };
 
-const Content = styled.div`
-  grid-area: content;
+const StyledBody = styled.div`
```

Comment:
It seems odd to use rows and cols everywhere else but a custom div here, what was the rationale?